app/Models/Product_Category.php

```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Product_Category extends Model
```

Comment:
Class names are UpperCamelCase
```suggestion
class ProductCategory extends Model
```